cmd/kubeadm/app/util/config/initconfiguration.go

```diff
@@ -155,6 +163,21 @@ func SetClusterDynamicDefaults(cfg *kubeadmapi.ClusterConfiguration, advertiseAd
 
 	// Downcase SANs. Some domain names (like ELBs) have capitals in them.
 	LowercaseSANs(cfg.APIServer.CertSANs)
+
+	if isDualStack {
```

Comment:
Yes, here I could use `ClusterConfiguration` since that is passed into the func; however, just to be consistent with the `SetNodeRegistrationDynamicDefaults` approach, I use `isDualStack`